### ISSUE TITLE
new boost version 1.83.0-8 breaks build

### ISSUE THREAD

**Issue by mrsteve0924** [KrystianD-contribution/easystroke#2]
any chance you can fix?  thanks

**Comment by KrystianD** [comment]
Hi, what error do you get? Just updated on my Arch with 1.83.0-8 and all works well (full rebuild)

**Comment by KrystianD** [comment]
Make sure to update gcc and glib as well. Reproduced the error after upgrading boost libraries only. Haven't checked full system update yet, but worth to try.

**Comment by mrsteve0924** [comment]
hi and thanks for the reply. 

just did a full system upgrade.  it pulled in an update to gcc, but still getting build error.
im on arch kernel 6.9.1-arch1-2
gcc version 14.1.1+r58+gfc9fb69ad62-1
glib2 version 2.80.2-2

this is the output of make
https://gist.github.com/mrsteve0924/893f4e8481e248f8f7d07d13fd21fddf

**Comment by mrsteve0924** [comment]
so are you able to build this without error?

**Comment by KrystianD** [comment]
Nope, haven't yet made a full system upgrade since that time, not planning for now.

**Comment by KrystianD** [comment]
should be fixed now

**Comment by mrsteve0924** [comment]
hello,  i keep getting these errors and fails to build

make
fatal: No annotated tags can describe '5774aaebc3e20e237ee48541fc58369eee939d04'.
However, there were unannotated tags: try --tags.
fatal: No annotated tags can describe '5774aaebc3e20e237ee48541fc58369eee939d04'.
However, there were unannotated tags: try --tags.
fatal: No annotated tags can describe '5774aaebc3e20e237ee48541fc58369eee939d04'.
However, there were unannotated tags: try --tags.
fatal: No annotated tags can describe '5774aaebc3e20e237ee48541fc58369eee939d04'

**Comment by KrystianD** [comment]
This doesn't look like anything in-repo related, try a fresh clone, just tried and all works well. Make sure to use `patched` branch

**Comment by mrsteve0924** [comment]
well even with those fatal errors it does build and install. so i guess im good.  thanks
